core/player.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, bullet_group, effects_group, life_board, score_board):
         self.hitrect_offset_y = -15
         self.score = 0
         self.lives = 3
+        #TODO: rework how player lives are set as magic numbers are used here and in the main game logic.
```

Comment:
Click the issues tab and create an issue if it is something you'd like to fix. Github uses issues as both bugs and feature requests.